### PR TITLE
Layers browser width calculated based on map width

### LINKED ISSUE #420: Browser Menu not displaying correct size and style
I realized what the problem is with the @media queries not working correctly in plots2: it's only checking the size of the *screen*, not the size of the map. If the map is only in the sidebar and is 200px wide the screen is still full size, so it's rendering based on that.

FireShot Capture 320 - 🎈 Public Lab_ Software Outreach - localhost

Same map, when screen itself is small, working correctly: 
FireShot Capture 319 - 🎈 Public Lab_ Software Outreach - localhost

I'm going to work on this from the plots2 end, but I think we need to have some additional styles for smaller sizes that we can apply and remove as needed. Then we can use javascript to check the width of the map itself.

### PR DESCRIPTION
**Pull request by nstjean**
Fixes #420 (<=== Add issue number here)

I've reformatted the styling of the menus as they shrink down in size so that the menu style applied is calculated based on the size of the map itself, not the browser window. This is done by dynamically adding a class to the `.leaflet-container`
Width 0-379px gets "xs"
width 380-589px gets "sm"
width 590-879px gets "md"
width 880+ gets "lg"

Unfortunately the width of the menu itself is a set width for each of the classes, as using a % width doesn't work correctly and it would take me more time to figure out a solution. But I think this offers a pretty reasonable experience at any map width!

`lg`
FireShot Capture 333 - Leaflet Environmental Layers -

`md`
FireShot Capture 334 - Leaflet Environmental Layers -

`sm`
FireShot Capture 335 - Leaflet Environmental Layers -

`xs`
FireShot Capture 336 - Leaflet Environmental Layers -

Peek 2020-03-02 15-18

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by crisner** [comment]
Does this work with browser resizing too like media queries does?

File: src/util/mapResize.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// set the map menu to the correct size
```
Comment (crisner) [inline src/util/mapResize.js]: What is this file for?

Comment (nstjean) [reply src/util/mapResize.js]: Whoops forgot to remove that. I ended up not using it.

**Comment by nstjean** [comment]
> Does this work with browser resizing too as media queries does?

Yes, as long as the map size changes then the menu will adjust its size.

**Comment by crisner** [comment]
I am curious. Does the map have to resize for it to take effect? Would it work as expected on smaller screens?

**Comment by nstjean** [comment]
No resize is needed, the function is called on load.

Phone size:
FireShot Capture 337 - Leaflet Environmental Layers -

And here's ipad size on example/index.html:
FireShot Capture 338 - Leaflet Environmental Layers -

**Comment by crisner** [comment]
Awesome!!! :tada:

**Pull request merged**